src/components/App.tsx

```diff
@@ -21,7 +23,22 @@ export const App = () => {
             path="/categories/:id"
             render={({match}) => <MoviesList genre={match.params.id} />}
           />
-          <Route path="/about" component={() => <h1>About</h1>} />
+          <Route
+            path="/about"
+            component={() => (
+              <h1
```

Comment:
I suggest extracting this to its own component with its styles in either `css` or `styled-components` for the sake of consistency.

Reply:
I will, in the future. When updating the readme files. I'll also try to  support `.md` file with `about`